src/api/__tests__/File-test.js

```diff
@@ -95,6 +97,76 @@ describe('api/File', () => {
         });
     });
 
+    describe('getThumbnailUrl()', () => {
+        const baseUrl = 'baseUrl';
+        const url_template = `${baseUrl}/{+asset_path}`;
+        const representation = 'jpg';
+
+        const baseItem = {
+            representations: {
+                entries: [
+                    {
+                        representation,
+                        status: { state: 'success' },
+                        content: {
+                            url_template,
+                        },
+                    },
+                ],
+            },
+        };
+
+        let item;
+        beforeEach(() => {
+            item = cloneDeep(baseItem);
```

Comment:
Is this the best way to do this? The properties I need to change in `item` are deeply nested, and I thought this would be cleaner than declaring a new item for every case.

Reply:
Another approach would be to have a `createTestItem` method that takes in arguments for what you want to change